src/shared/sandboxCommandBase.ts

```diff
@@ -230,6 +237,32 @@ export abstract class SandboxCommandBase<T> extends SfCommand<T> {
     }
   }
 
+  // Gets the SandboxName either from the request data or the SandboxProcessObject
+  protected getSandboxName(): string | undefined {
+    return this.sandboxRequestData?.sandboxProcessObject.SandboxName ?? this.latestSandboxProgressObj?.SandboxName;
+  }
+
+  // Not sure why the lint rule is complaining since it's definitely called.
+  // eslint-disable-next-line class-methods-use-this
```

Comment:
it's because the method doesn't use `this` in it

Reply:
Ah, that's right, thank you!